vdev/src/commands/meta/install_git_hooks.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::{Result, Ok};
+use std::fs::File;
+use std::io::Write;
+use std::os::unix::fs::PermissionsExt;
```

Comment:
Posting what was discussed in 1:1 with Bruce:

Add a configuration conditional checks around the import statement